frontend/package.json

```diff
@@ -26,7 +26,7 @@
     "vue-i18n-extract-remove": "vue-i18n-extract --remove"
   },
   "dependencies": {
-    "@jellyfin/sdk": "0.0.0-unstable.202301101857",
+    "@jellyfin/sdk": "0.7.0",
```

Comment:
I'll remove this commit for now, thanks for letting me know. There's only one real bug filed in that repo right now. Does that mean the next release should be a good time to upgrade, or are there other issues you're aware of that aren't filed which would prevent that?

Reply:
@ThibaultNocchi bumping my above question. I can merge without it answered, but I'm curious if there's any other issues that aren't filed which we can't upgrade because of?